ms/MSOper/MSMetaData.cc

```diff
@@ -1428,6 +1428,13 @@ std::map<ScanKey, std::set<uInt> > MSMetaData::getScanToSpwsMap() const {
     return scanToSpwMap;
 }
 
+vector<std::set<ScanKey> > MSMetaData::getSpwToScansMap() const {
```

Comment:
spw's here are always indexed 0 to vector.size()?
I would have expected a map int -> set (or multi_map)